roll/http.py

```diff
@@ -31,9 +30,15 @@ class HttpError(Exception):
 
     __slots__ = ('status', 'message')
 
-    def __init__(self, http_code: HttpCode, message: str=None):
+    def __init__(self, http_code: HttpCode, message: str=None, context: Exception=None):
```

Comment:
I've hesitated between changing HttpError signature and instead creating a shortcut like `HttpError.from_error(error)`

Reply:
The current implementation looks good to me. `HttpError.from_error(error)` contains too much errors 🙃